cmd/jira-lifecycle-plugin/server.go

```diff
@@ -1826,8 +1826,11 @@ refIssueLoop:
 		// GetIssue is invalid for setting the field ourselves
 		sprintField := bug.Fields.Unknowns[helpers.SprintField]
 		delete(bug.Fields.Unknowns, helpers.SprintField)
-		releaseNoteType := bug.Fields.Unknowns[helpers.ReleaseNoteTypeField]
-		releaseNoteText := bug.Fields.Unknowns[helpers.ReleaseNoteTextField]
```

Comment:
Did you mean to delete the lines associated with ReleaseNotes?

Reply:
Not sure how that happened (I think I may have checked out an older version of server.go before the release note changes?). Should be good now